.template/ios/Podfile

```
@@ -37,5 +37,59 @@ end
 post_install do |installer|
   installer.pods_project.targets.each do |target|
     flutter_additional_ios_build_settings(target)
+    target.build_configurations.each do |config|
+              # You can enable the permissions needed here. For example to enable camera
```

Can we start this with a TODO comment? + No example here

```suggestion
              # TODO You can enable the permissions needed here.
```